### PR TITLE
[UD] fixes for "Get More RAM" widget and Billing info tab

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?
This PR adds fixes for Billing info tab and "Get More RAM" widget:
- restrict some inputs to enter only valid symbols;
- set focus to input when widget is shown;
- apply filter to round fractional part of calculated values in widget;

### What issues does this PR fix or reference?
N/A

#### Changelog

[UD] make inputs on "Get More RAM" widget and Billing info tab to allow to enter only valid symbols

#### Release Notes

N/A - bugfix

#### Docs PR

N/A - bugfix

Signed-off-by: Oleksii Kurinnyi <okurinnyi@codenvy.com>

### CODE REVIEW

File: dashboard/src/app/billing/card-info/card-info.html [hunk 1]
```diff
@@ -51,6 +52,7 @@
                che-label-name="State"
                che-name="state"
                che-place-holder=""
+               che-type-city
```
Comment (benoitf) [inline dashboard/src/app/billing/card-info/card-info.html]: why do we have validator of cities for the state ?

Comment (akurinnoy) [inline dashboard/src/app/billing/card-info/card-info.html]: It's not validator of cities, it is just directive which prevent to enter not valid symbols to input.

Comment (benoitf) [inline dashboard/src/app/billing/card-info/card-info.html]: ok but my issue is that the name of validator contains "city" while it is more generic

Comment (akurinnoy) [inline dashboard/src/app/billing/card-info/card-info.html]: Could you suggest more suitable name?

Comment (benoitf) [inline dashboard/src/app/billing/card-info/card-info.html]: @akurinnoy well this change is not urgent I approved the PR. I don't have more suitable name. Maybe che-type-city and che-type-state directives that can be based on the same class.

Comment (akurinnoy) [inline dashboard/src/app/billing/card-info/card-info.html]: Thanks! I will update it soon.

**Pull request merged**